src/components/SelectionList/TransactionListItem.tsx

```diff
@@ -108,6 +137,22 @@ function TransactionListItem<TItem extends ListItem>({
         );
     };
 
+    const categoryCell = (
```

Comment:
isn't something like this simpler?

```js
const categoryCell = isLargeScreenWidth ? (
   <big version...
) : (
   <small version);
```

I find the definition of both `category|tagCell` and `smallCategory|tagCell` a bit confusing.

Reply:
I like @Kicu's suggestion better than splitting this logic in two separate places.